src/quo2/components/buttons/button/style.cljs

```diff
@@ -103,4 +103,4 @@
            {:border-color border-color
             :border-width 1})
          (when disabled
-           {:opacity 0.3})))
+           {:opacity 0.7})))
```

Comment:
why did you do this? these styles were set correctly- button is used on pretty every page so it's best to reach out to be sure if there's some changes like this needed